app/services/zendesk_intake_service.rb

```diff
@@ -21,6 +21,7 @@ def instance_eitc?
 
   def create_intake_ticket_requester
     # returns the Zendesk ID of the created user
+    # TODO: move notification opt in columns to intake from user so contact info can also move
```

Comment:
this would require more changes to the notification page. also potentially a backfill of the opt-in columns if we are too move the column from user to intake?